src/lib/tvgShape.cpp

```diff
@@ -288,6 +288,14 @@ const Fill* Shape::fill() const noexcept
 }
 
 
+Result Shape::order(bool strokeFirst) noexcept
+{
+    if (strokeFirst && !pImpl->strokeFirst()) return Result::FailedAllocation;
```

Comment:
Please take cover the strokeFirst == false scenario.

Reply:
done